### DISCUSSION TITLE
Transfer products between 2 bagisto project

### DISCUSSION THREAD

**Discussion by aghaeian** [bagisto/bagisto#10233] [Q&A]
Hello,
I want to transfer created products between 2 different bagisto project. 
How i can do this?

Best Regards.

**Comment by suraj-webkul** [comment]
Hello @aghaeian,

Currently, there isn't a built-in method in Bagisto to transfer products directly between two projects. However, you can achieve this by exporting the product-related tables from the database of the source project and importing them into the destination project. 

Additionally, you’ll need to copy the `storage` directory, which contains product images and other media files, from the source project to the destination project. Please ensure that both projects are on compatible versions to avoid conflicts.

As it’s important to take a backup of both the database and the codebase of each project before proceeding with this process.

Let me know if you need further assistance!

Best regards,  
Suraj
